src/Core/Services/StateMigrationService.cs

```diff
@@ -544,6 +547,296 @@ private class V5Keys
+                    await MoveToPrefsAsync<string>(V6Keys.EncPrivateKeyKey(userId));
+                    await MoveToPrefsAsync<List<string>>(V6Keys.AutofillBlacklistedUrisKey(userId));
+                    await MoveToPrefsAsync<long?>(V6Keys.LastActiveTimeKey(userId));
+                    await MoveToPrefsAsync<int?>(V6Keys.VaultTimeoutKey(userId));
+                    await MoveToPrefsAsync<V6Keys.VaultTimeoutAction?>(V6Keys.VaultTimeoutActionKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.ScreenCaptureAllowedKey(userId));
+                    await MoveToPrefsAsync<V6Keys.PreviousPageInfo>(V6Keys.PreviousPageKey(userId));
+                    await MoveToPrefsAsync<int>(V6Keys.InvalidUnlockAttemptsKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.DisableAutoTotpCopyKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.InlineAutofillEnabledKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.AutofillDisableSavePromptKey(userId));
+                    await MoveToPrefsAsync<int?>(V6Keys.DefaultUriMatchKey(userId));
+                    await MoveToPrefsAsync<int?>(V6Keys.ClearClipboardKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.PasswordRepromptAutofillKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.PasswordVerifiedAutofillKey(userId));
+                    await MoveToPrefsAsync<DateTime?>(V6Keys.LastSyncKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.SyncOnRefreshKey(userId));
+                    await MoveToPrefsAsync<DateTime?>(V6Keys.PushLastRegistrationDateKey(userId));
+                    await MoveToPrefsAsync<string>(V6Keys.PushCurrentTokenKey(userId));
+                    await MoveToPrefsAsync<Dictionary<string, V6Keys.PolicyData>>(V6Keys.PoliciesKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.UsesKeyConnectorKey(userId));
+                    await MoveToPrefsAsync<Dictionary<string, V6Keys.OrganizationData>>(
+                        V6Keys.OrganizationsKey(userId));
+                    await MoveToPrefsAsync<V6Keys.PasswordGenerationOptions>(V6Keys.PassGenOptionsKey(userId));
+                    await MoveToPrefsAsync<V6Keys.UsernameGenerationOptions>(V6Keys.UsernameGenOptionsKey(userId));
+                    await MoveToPrefsAsync<string>(V6Keys.TwoFactorTokenKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.ApprovePasswordlessLoginsKey(userId));
+                    await MoveToPrefsAsync<bool?>(V6Keys.ShouldConnectToWatchKey(userId));
+                    await MoveToPrefsAsync<string>(V6Keys.DeviceKeyKey(userId));
+                }
+            }
+
+            await RemoveValueAsync(Storage.LiteDb, Constants.StateKey);
+            await RemoveValueAsync(Storage.LiteDb, V6Keys.LastActiveTimeKey(""));
+
+            // Update stored version
+            await SetLastStateVersionAsync(5);
```

Comment:
⛏️ Shouldn't this be `6`?
```suggestion
            await SetLastStateVersionAsync(6);
```

Reply:
Yikes, good catch.